src/librustc_codegen_llvm/coverageinfo/mod.rs

```diff
@@ -0,0 +1,60 @@
+use crate::common::CodegenCx;
+use log::debug;
+use rustc_codegen_ssa::traits::CoverageInfoMethods;
+use rustc_hir::def_id::LOCAL_CRATE;
+
+// FIXME(richkadel): This might not be where I put the coverage map generation, but it's a
+// placeholder for now.
+//
+// I need to inject the coverage map into the LLVM IR. This is one possible place to do it.
+//
+// The code below is here, at least temporarily, to verify that the `CoverageRegion`s are
+// available, to produce the coverage map.
+
+/// Generates and exports the Coverage Map.
+pub fn finalize(cx: &CodegenCx<'_, '_>) {
+    let instances = &*cx.instances.borrow();
+    for instance in instances.keys() {
+        if instance.def_id().krate == LOCAL_CRATE {
+            // FIXME(richkadel): Is this check `krate == LOCAL_CRATE` right?
+            //
+            // NOTE: I'm considering at least one alternative to this loop on `instances`,
+            // But if I do go with this, make sure the check on LOCAL_CRATE works for all cases.
```

Comment:
See my [PR comment](https://github.com/rust-lang/rust/pull/73684#issuecomment-649882503) regarding an alternative approach I'm considering.

If I go with the alternative approach, ignore the implementation of this function and the comments.